ui/app/AppLayouts/Chat/panels/communities/CommunityMintTokensSettingsPanel.qml

```diff
@@ -244,4 +269,28 @@ SettingsPageLayout {
             }
         }
     }
+
+    Component {
+        id: signTransactionDialog
+
+        SignMintTokenTransactionPopup {
+            id: popup
+
+            onOpened: root.signMintTransactionOpened()
+            onCancelClicked: close()
+            onSignTransactionClicked: root.signMintTransactionClicked()
```

Comment:
is it necessary to  have that `root.signMintTransactionClicked` in that component api?

Reply:
Here somehow we need to start the minting flow, but we need to go back to the stackview component that has the information / parameters needed to start the minting. By this signal we can do that. 
Another alternative could be adding a global param and add a binding and listen to the property change inside the stackview component in order to start the flow.
I thought it is clearer using the signal that way.